draft-ietf-rtcweb-jsep.xml

```diff
@@ -2668,6 +2698,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                   layers have been established. Otherwise, stop transmitting RTP media,
                   although RTCP should still be sent, as described in 
                   <xref target="RFC3264"/>, Section 5.1. </t>
+		<t>If RID values are specified, include the RID header
```

Comment:
I think we need some discussion in the 'Verification' section above ensuring that all RIDs referenced in simulcast appear as a=rid lines.